scripts/.gitignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
```suggestion
*.js

```

Reply:
Are there lots of `.js` files being ignored? isn't it just `scripts/embassy.js`? if so you can just put this in the parent dir `.gitignore`